README.md

```diff
@@ -85,12 +85,47 @@ TEST_CASE("Parsing hello world program")
 ### Get the source
 
 ```sh
-git clone https://github.com/satya-das/common.git
-git clone https://github.com/satya-das/CppParser.git
+git clone https://github.com/agladilin/cpppqqarser.git
 ```
 
 ## Configure and build
 
+### Windows Visual Studio
+
+[Install vcpkg](https://vcpkg.io/en/getting-started) to fetch boost
+
+```powershell
+vcpkg install boost-filesystem boost-program-options boost-system boost-date-time
+cd cppparser
+mkdir build
+cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE="[path to vcpkg]/scripts/buildsystems/vcpkg.cmake"
+#cd build
+#msbuild cppparser.sln
+cmake --build build --config Debug
+.\build\Debug\cppparsertest.exe
+.\build\Debug\cppparserunittest.exe
+cmake --build build --config Debug --target install  # requires administrator privileges
+vcpkg install cppparser --overlay-ports=c:\dev\custom-overlay
+```
+
+To remove
+
+```powershell
+vcpkg remove cppparser
+del $VCPKG_ROOT\buildtrees\cppparser
+del $VCPKG_ROOT\packages\cppparser_x64-windows
+del $VCPKG_ROOT\downloads\agladilin-cppparser-0.1.0.tar.gz
+del C:\Users\<user>\AppData\Local\vcpkg\archives\19 # or whatever corresponding to package ABI hash in install log
+```
+
+### Linux and others
+
+*For Ubuntu to fetch boost:*
+
+```sh
+sudo apt install libboost-dev libboost-system-dev libboost-filesystem-dev libboost-program-options-dev
```

Comment:
I followed this process and I get following error:
```
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Boost-1.74.0/BoostConfig.cmake:141 (find_package):
  Could not find a package configuration file provided by "boost_date_time"
  (requested version 1.74.0) with any of the following names:

    boost_date_timeConfig.cmake
    boost_date_time-config.cmake

  Add the installation prefix of "boost_date_time" to CMAKE_PREFIX_PATH or
  set "boost_date_time_DIR" to a directory containing one of the above files.
  If "boost_date_time" provides a separate development package or SDK, be
  sure it has been installed.
Call Stack (most recent call first):
  /usr/lib/x86_64-linux-gnu/cmake/Boost-1.74.0/BoostConfig.cmake:258 (boost_find_component)
  /usr/local/pkg/share/cmake-3.26/Modules/FindBoost.cmake:594 (find_package)
  CMakeLists.txt:29 (find_package)
```
I had tried to remove the `common` repository before by using things like this. But unfortunately it creates build issues one way or the other.
I will be happy to take an improvement like this if it builds seamlessly.